src/Misc/layoutroot/run.sh

```diff
@@ -43,8 +43,6 @@ else
         else
             sleep 5 >nul
         fi
-        
-        "$DIR"/bin/Runner.Listener run $*
```

Comment:
why we don't start the runner again after update?

Reply:
Well, what's interesting the runner is restarted forked without requiring this, moreover the script exits!
As stated `Runner will exit shortly for update, should back online within 10 seconds.`

Pay attention to the last line where I'm already back in terminal the parent has exited as well as the run.sh
```
2021-02-04 13:07:56Z: Listening for Jobs
Runner update in progress, do not shutdown runner.
Downloading 2.276.1 runner
Waiting for current job finish running.
Generate and execute update script.
Runner will exit shortly for update, should back online within 10 seconds.
+ returnCode=3
+ [[ 3 == 4 ]]
+ exit 3
root@github-actions-deployer-pool-pod-44wd6:/home/runner# 
√ Connected to GitHub
```

Reply:
This is hot automatic update performed by the runner itself, we need only to wait for the update process finish, so that we don't break unfinished upgrade if the container which spawned run.sh is terminated immediately.